python_ta/cfg/visitor.py

```diff
@@ -55,7 +56,7 @@ def __getattr__(self, attr: str):
         if attr.startswith("visit_"):
             return self.visit_generic
         else:
-            raise AttributeError(f"'CFGVisitor' object has not attribute '{attr}'")
+            raise AttributeError(f"'CFGVisitor' object has no attribute '{attr}'")
```

Comment:
Fixes minor typo in error message. Is this an appropriate change to include in this PR?

Reply:
It's a good catch, but yeah that should be in a separate PR.